src/basic_recipes/error_and_rangebars.jl

```diff
@@ -261,8 +261,9 @@ function plot_to_screen(plot, points::AbstractVector)
     spvm = clip_to_space(cam, :pixel) * space_to_clip(cam, space) *
         f32_convert_matrix(plot, space) * transformationmatrix(plot)[]
 
+    transfunc = transform_func(plot)
```

Comment:
here, transfunc should be passed in to plot_to_screen, lifted from transform_func_obs(plot) here:

https://github.com/MakieOrg/Makie.jl/pull/4999/files#diff-66312870738e99091c1bc0a5c6577921970f5caabba3348970fefdef0b963369L212